### PR TITLE
Functional tests - Change selectors for catalog pages

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Change selectors for catalog pages
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | ` DOWNLOAD_PATH='your_dowload_path' TEST_PATH="functional/BO/03_catalog/*" URL_FO=shopUrl/ npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18849)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/puppeteer/pages/BO/catalog/brands/index.js [hunk 1]
```diff
@@ -14,59 +14,55 @@ module.exports = class Brands extends BOBasePage {
...
-      .replace('%TABLE', 'manufacturer');
+    this.brandsTableColumnLogoImg = row => `${this.tableColumn('manufacturer', row, 'logo')} img`;
+    this.brandsTableEnableColumn = row => `${this.tableColumn('manufacturer', row, 'active')}`;
+    this.brandsEnableColumnValidIcon = row => `${this.brandsTableEnableColumn(row)} i.grid-toggler-icon-valid`;
+    this.brandsEnableColumnNotValidIcon = row => `${this.brandsTableEnableColumn(row)} i.grid-toggler-icon-not-valid`;
+    this.viewBrandLink = row => `${this.actionsColumn('manufacturer', row)} a[data-original-title='View']`;
+    this.editBrandLink = row => `${this.dropdownToggleMenu('manufacturer', row)} a[href*='/edit']`;
+    this.bulkActionsEnableButton = `${this.gridPanel('manufacturer')} #manufacturer_grid_bulk_action_enable_selection`;
+    this.bulkActionsDisableButton = `${this.gridPanel('manufacturer')
+    } #manufacturer_grid_bulk_action_disable_selection`;
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/catalog/brands/index.js:58]: Why this end of line ?

Comment (SimonGrn) [reply tests/puppeteer/pages/BO/catalog/brands/index.js:58]: Linter did this...

Comment (Progi1984) [reply tests/puppeteer/pages/BO/catalog/brands/index.js:58]: Bad linter.

Comment (NeOMakinG) [reply tests/puppeteer/pages/BO/catalog/brands/index.js:58]: Isn't a rule with length of lines ? that could be adjusted

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/catalog/brands/index.js:58]: we already adjust it to 120 characters, i think it's enough.

**Review by matthieu-rolland — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**